src/api/files.js

```diff
@@ -10,8 +10,12 @@ import { imagesDir } from 'utils/args';
 
 const { host, token } = config.services.fibery;
 
-const filesProxy = async ({ params: { secret }, query: { w = '', f = 'png' } }, res, next) => {
-  const filepath = `${imagesDir}/${secret}_${w}.${f}`;
+const filesProxy = async (
+  { params: { secret }, query: { w = '', h = '', f = 'png' } },
+  res,
+  next
+) => {
+  const filepath = `${imagesDir}/${secret}_${w}x${h}.${f}`;
```

Comment:
what if `h` is passed but `w` is not? `x` still will be a marker on how the image was resized?

Reply:
yep